README.md

```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+#### Biography (HW1) 
+
+Hi, 
+
+My name is **Victor Vazquez**. I am a second year MPP student from ~~Haris Schul~~, sorry, from Harris School of Public Policy. I have work experience in the private sector, especially in the banking sector. I am a current member of the organizations *Latin American Matters (LAM)* and *Mexicans at Uchicago (UCMA)* at the University of Chicago. Some of the events we like to organize are:
+
+- Discussions with speakers from around the world 
+- Wine Tasting
+    - Includes some games to meet each other
+- Trips to latinamerican countries
+
+One of the links of LAM is [here](https://lamuchicago.wordpress.com/).
+
+![LAM logo](/Users/victorhugovazquez/Desktop/lam.jpg)
+![Victor](/Users/victorhugovazquez/Desktop/picture.jpg)
+
```

Comment:
The images didn't render properly.  I think it's because you used a local file path.

Reply:
Yes, that is why the pictures file to load. Try using a relative file path next time.